analyzers/rspec/vbnet/S1481.html

```diff
@@ -5,16 +5,16 @@ <h2>Why is this an issue?</h2>
 <h3>What is the potential impact?</h3>
 <p>Having unused local variables in your code can lead to several issues:</p>
 <ul>
-  <li> Decreased Readability: Unused variables can make your code more difficult to read. They add extra lines and complexity, which can distract from
-  the main logic of the code. </li>
-  <li> Misunderstanding: When other developers read your code, they may wonder why a variable is declared but not used. This can lead to confusion and
-  misinterpretation of the code’s intent. </li>
-  <li> Potential for Bugs: If a variable is declared but not used, it might indicate a bug or incomplete code. For example, if you declared a variable
-  intending to use it in a calculation, but then forgot to do so, your program might not work as expected. </li>
-  <li> Maintenance Issues: Unused variables can make code maintenance more difficult. If a programmer sees an unused variable, they might think it is
-  a mistake and try to 'fix' the code, potentially introducing new bugs. </li>
-  <li> Memory Usage: Although modern compilers are smart enough to ignore unused variables, not all compilers do this. In such cases, unused variables
-  take up memory space, leading to inefficient use of resources. </li>
+  <li> <strong>Decreased Readability</strong>: Unused variables can make your code more difficult to read. They add extra lines and complexity, which
```

Comment:
This change was done by @jonas-wielage-sonarsource for IaC in the context of https://github.com/SonarSource/rspec/pull/3790.
The change is only about formatting, and it's OK for both C# and VB.NET.